modules/phase_field/src/base/PhaseFieldApp.C

```diff
@@ -236,10 +236,10 @@
 /*
  * VectorPostprocessors
  */
- #include "GrainCentersPostprocessor.h"
- #include "GrainForcesPostprocessor.h"
- #include "GrainTextureVectorPostprocessor.h"
-
+#include "GrainCentersPostprocessor.h"
+#include "GrainForcesPostprocessor.h"
+#include "GrainTextureVectorPostprocessor.h"
+#include "FeatureVolumeVectorPostprocessor.h"
```

Comment:
We order those (and the register calls alphabetically - except deprecated registrations, which are at the end of each block)